src/pages/VideoPlayerPage.jsx

```diff
@@ -1,21 +1,23 @@
 import React from "react";
+import { useEffect } from "react";
```

Comment:
Can write this and line number 1 in a single line
import React ,{useEffect} from "react";

Reply:
noted